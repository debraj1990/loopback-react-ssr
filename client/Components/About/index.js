import React from 'react'

const styles = {
    display: 'flex',
    justifyContent:'center'
}

const About = () => (
  <div>
    <h1 style={styles} >Hi, I am Terrence!</h1>
    <p  style={styles}>This section is all about me, yay!</p>
  </div>
)

export default About
