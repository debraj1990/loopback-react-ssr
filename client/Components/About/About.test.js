import React from 'react';
import {configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16';

import About from './'


configure({adapter: new Adapter });


describe('About page', () => {
  test('About component should render as expected', () => {
    const component = shallow(<About />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  });
})