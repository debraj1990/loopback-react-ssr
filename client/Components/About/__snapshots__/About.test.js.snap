// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page About component should render as expected 1`] = `
<div>
  <h1
    style={
      Object {
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    Hi, I am Terrence!
  </h1>
  <p
    style={
      Object {
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    This section is all about me, yay!
  </p>
</div>
`;
